config/configgrpc/configgrpc_test.go

```diff
@@ -536,3 +536,27 @@ func tempSocketName(t *testing.T) string {
 	require.NoError(t, os.Remove(socket))
 	return socket
 }
+
+func TestGRPCPortReuse(t *testing.T) {
+	settings := GRPCServerSettings{
+		NetAddr: confignet.NetAddr{
+			Endpoint:  "localhost: 2873",
+			Transport: "tcp",
+		},
+	}
+	muxListener, err := settings.ToListener()
+	if err != nil {
+		t.Errorf("cannot initialize multiplexing listener")
+	}
```

Comment:
I'm not 100% sure what I'm saying is right, but maybe this should be changed to something like
```suggestion
	assert.NoError(t, err)
```
or if you want the test to terminate here, which probably (?) seems to be right:
```suggestion
	require.NoError(t, err)
```
This seems like what's been done in the other tests in the file, since I guess you don't need to print the exact error (it will tell you anyway). However, not sure which of `assert` or `require` is preferred here...

Reply:
Hey Eddy, I think we should print the error here because the `err` returned from `ToListener` method would not be the same.